streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -942,21 +943,20 @@ public void close() {}
         config.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
         config.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.Long().getClass().getName());
 
-        {
-            final TopologyTestDriver testDriver = new TopologyTestDriver(topology, config);
+        try (final TopologyTestDriver testDriver = new TopologyTestDriver(topology, config)) {
             Assert.assertNull(testDriver.getKeyValueStore("storeProcessorStore").get("a"));
             testDriver.pipeInput(recordFactory.create("input-topic", "a", 1L));
             Assert.assertEquals(1L, testDriver.getKeyValueStore("storeProcessorStore").get("a"));
-            testDriver.close();
         }
 
-        {
-            final TopologyTestDriver testDriver = new TopologyTestDriver(topology, config);
+
+        try (final TopologyTestDriver testDriver = new TopologyTestDriver(topology, config)) {
             Assert.assertNull(
-                "Closing the prior test driver should have cleaned up this store and value.",
-                testDriver.getKeyValueStore("storeProcessorStore").get("a")
+                    "Closing the prior test driver should have cleaned up this store and value.",
+                    testDriver.getKeyValueStore("storeProcessorStore").get("a")
```

Comment:
Nit: avoid unnecessary reformatting. Also, we use 4 space indention, not 8 space.

Reply:
Fixed it, my IDE was set to 8 spaces for "continuation indent".
Is there any official style guide or export of accepted settings?

Reply:
> Is there any official style guide or export of accepted settings?

Not really :(

Rule of thumb is, avoid reformatting if possible :)

Reply:
@mjsax Should we include indentation settings here: http://kafka.apache.org/coding-guide.html ? It's mentioned in the scala guidelines...

Reply:
Feel free to do a PR against https://github.com/apache/kafka-site